src/Dhadgar.Scope/src/components/ui/TextField.tsx

```
@@ -1,64 +1,64 @@
+  value,
+  onChange,
+  placeholder,
+  label,
+  clearable = false,
+  onClear,
+  className,
+  icon,
+}: TextFieldProps) {
+  return (
+    <div className={clsx("relative", className)}>
+      {label && <label className="block text-sm text-white/60 mb-1">{label}</label>}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)